### PR TITLE
[code refactor]merge torch_blade and torch_disc library

### LINKED ISSUE #396: [code refactor]merge torch_blade.so and torch_disc.so
The current codebase released two library: `torch_blade.so` and `torch_disc.so`, for better user experience, we should keep only one dynamic library `torch_blade.so`.  we should merge some bazel configuration and code from `torch_disc` into `torch_blade`.

### PR DESCRIPTION
**Pull request by Yancey1989**
fixed #396

### CODE REVIEW

File: pytorch_blade/src/ltc/disc_backend/backend_impl.cpp [hunk 1]
```diff
@@ -269,6 +269,8 @@ torch::lazy::BackendImplInterface* GetDISCBackendImpl() {
 
 void InitTorchScriptBackend() {
   static std::unique_ptr<torch::lazy::BackendRegistrar> s_registrar;
+  std::cout << "Welcome to Disc accelerator based on PyTorch LazyTensor Core!!!"
```
Comment (fortianyou) [inline pytorch_blade/src/ltc/disc_backend/backend_impl.cpp:272]: Nit: use logger?

File: pytorch_blade/src/ltc/init_python_bindings.cpp [hunk 2]
```diff
@@ -16,10 +16,8 @@
 namespace torch_disc {
 namespace py = pybind11;
 void InitLtcModuleBindings(py::module m) {
-  m.def("_ltc_init_disc_backend", []() { compiler::InitTorchScriptBackend(); });
-}
-
-PYBIND11_MODULE(_torch_disc, m) {
-  torch_disc::InitLtcModuleBindings(m);
+  py::module ltc =
+      m.def_submodule("_ltc", "torch_blade python bindings to mlir");
```
Comment (fortianyou) [inline pytorch_blade/src/ltc/init_python_bindings.cpp:20]: Nit: python bindings to lazy tensor core?

File: pytorch_blade/src/ltc/disc_backend/backend_impl.cpp (hunk 1 above)
Comment (Yancey1989) [reply pytorch_blade/src/ltc/disc_backend/backend_impl.cpp]: done

File: pytorch_blade/src/ltc/init_python_bindings.cpp (hunk 2 above)
Comment (Yancey1989) [reply pytorch_blade/src/ltc/init_python_bindings.cpp]: done

File: pytorch_blade/torch_blade/__init__.py [hunk 3]
```diff
@@ -9,6 +9,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from ast import Raise
```
Comment (fortianyou) [inline pytorch_blade/torch_blade/__init__.py:12]: Nit: is this used?

File: pytorch_blade/torch_blade/__init__.py [hunk 4]
```diff
@@ -22,6 +23,7 @@
 import torch_blade.utils as utils
 import torch_blade.version as version
 import warnings
+import importlib
```
Comment (fortianyou) [inline pytorch_blade/torch_blade/__init__.py:26]: ditto

**Review by fortianyou — APPROVED** [review]
LGTM

File: pytorch_blade/torch_blade/__init__.py (hunk 3 above)
Comment (Yancey1989) [reply pytorch_blade/torch_blade/__init__.py:12]: will remove these in the next PR.

**Pull request merged**